### DISCUSSION TITLE
Setting MOTD and image to server list on minestom

### DISCUSSION THREAD

**Discussion by dev-hydrogen** [Minestom/Minestom#1210] [Help] [answered]
I'm pretty sure I should just listen to [ServerListPingEvent](https://javadoc.minestom.net/net/minestom/server/event/server/ServerListPingEvent.html), and use setResponseData, however i'd just like to confirm this (and add it to the discussions bank 😛)

**Comment by LooFifteen** [comment, marked as answer]
Yes, you are correct.
